tasks.ts

```diff
@@ -54,7 +56,8 @@ export const taskMiddleware = store => next => action => {
   if (tasks.length > 0) {
     throw lastWithTaskCall;
   }
-  const returnValue = next(action);
+
+  next(action);
   const dispatch = store.dispatch;
```

Comment:
You could move the call to `ASYNC_DISPATCH` here so you aren't re-creating the wrapped `dispatch` for each task.